tpl/template_funcs.go

```diff
@@ -1803,7 +1803,7 @@ func htmlUnescape(in interface{}) (string, error) {
 
 func init() {
 	funcMap = template.FuncMap{
-		"absURL":       func(a string) template.HTML { return template.HTML(helpers.AbsURL(a)) },
+		"absURL":       func(a interface{}) template.HTML { return template.HTML(helpers.AbsURL(cast.ToString(a))) },
```

Comment:
Please use the ToStringE variant and return any error as second return value from the template func. Same with relURL.